### PR TITLE
Minor copyediting

### PR DESCRIPTION
**Pull request by ahaynes-ador**
:green_book: 

Kathy can thumb and merge at will.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # LMHT - HTML: Lenguaje de Marca de Hipertexto - Hyper Text Markup Language
 
-Welcome to [LMHT].  This is an open source code repository to help Spanish speakers translate and understand HTML tags.  Originally started as a project by Team Open Brace in the [YWebCA] Summer of 2015 class, this project is now being developed further as an open source project by [Adorable] to expand it's possibilities.  With your help, we can expand this into an excellent learning tool for bilingual programmers.  
+Welcome to [LMHT].  This is an open source code repository for a website that helps Spanish speakers translate and understand HTML tags. Originally started as a project by Team Open Brace in the [YWebCA] Summer 2015 cohort, this project is now being developed further by [Adorable] as an open source project to expand its possibilities. With your help we can expand this into an excellent learning tool for bilingual programmers.
```
Comment (ericanaughton) [inline README.md:3]: @LipGlossary or @kathyoleary why are there brackets around Adorable? just wonderin

**Comment by ericanaughton** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (ahaynes-ador) [inline README.md:3]: They're citation-style links, defined on lines 5 and 6.

Comment (ericanaughton) [inline README.md:3]: okie dokes

**Comment by JenEEsquivel** [comment]
"Open Source" or "open source" in lines 5, line 25, etc.?   I believe the words are usually in caps.  There are a few instances where I saw they were lower-case in document.  @LipGlossary, should they always be caps? (@kathyoleary)

**Comment by ahaynes-ador** [comment]
My sense is that using title case refers to open sourcing as a paradigm. I think they can both be appropriate depending on the context. But consistency is good.

File: README.md [hunk 2]
```diff
@@ -8,23 +8,21 @@ Welcome to [LMHT].  This is an open source code repository to help Spanish speak
 
 ## Introduction
 
-The [LMHT page] provides a list of current HTML tags.  Select the tag you are looking for or begin typing it in the search box.  Once selected, a separate window will pop up containing the english base of the tag, the spanish equivalent, a definition in both english and spanish as well as audio recordings of each of these items.  Furthermore, there are helpful hints on how to remember what the tag does as well as examples of the code being used.
+The [LMHT site] provides a list of current HTML tags. Select the tag you are looking for or begin typing it in the search box. Once selected, an embedded window will appear, containing the English base of the tag, the Spanish equivalent, a definition in both English and Spanish, as well as audio recordings of each of these items.  Furthermore, there are helpful hints on how to remember what the tag does as well as examples of the code being used.
 
 [LMHT page]:  http://lmht.github.io/
```
Comment (kathyoleary) [inline README.md]: This needs to be changed from [LMHT page] to [LMHT site] based on the changes submitted.

**Comment by kathyoleary** [comment]
Added a line note about the change needed.  I had originally submitted PR #2 not realizing that the conditional thumbs up meant changes had to be updated before it was an ok.  So merged it to master without doing those.  Annalise put through those changes after I merged it since I didn't, but I have a change that has to be done based on her changes.  Put the line note in on the change and flowdocked  her to see if she can resubmit to master or back to this branch so we can get it done.

**Comment by kathyoleary** [comment]
submitted the changes myself and sent out request for double thumbs up

**Comment by rylnd** [comment]
:+1:

**Comment by ericanaughton** [comment]
@bigtiger  said this is good to go. :+1:

**Pull request merged**